### PR TITLE
tweak(rnd) Add prefix in logs

### PR DESCRIPTION
**Pull request by aarushik93**
### PR Quality Scorecard ✨

- [x] Have you used the PR description template? &ensp; `+2 pts`
- [ ] Is your pull request atomic, focusing on a single change? &ensp; `+5 pts`
- [ ] Have you linked the GitHub issue(s) that this PR addresses? &ensp; `+5 pts`
- [ ] Have you documented your changes clearly and comprehensively? &ensp; `+5 pts`
- [ ] Have you changed or added a feature? &ensp; `-4 pts`
  - [ ] Have you added/updated corresponding documentation? &ensp; `+4 pts`
  - [ ] Have you added/updated corresponding integration tests? &ensp; `+5 pts`
- [ ] Have you changed the behavior of AutoGPT? &ensp; `-5 pts`
  - [ ] Have you also run `agbenchmark` to verify that these changes do not regress performance? &ensp; `+10 pts`

___

### **PR Type**
enhancement

___

### **Description**
- Introduced a new function `get_log_prefix` to generate consistent log prefixes for better traceability.
- Updated various logging statements throughout the `manager.py` file to include the newly generated log prefixes.
- Modified function signatures to accept and utilize log prefixes, enhancing the clarity of log messages.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>manager.py</strong><dd><code>Add log prefixes to enhance log clarity and traceability</code>&nbsp; </dd></summary>
<hr>

rnd/autogpt_server/autogpt_server/executor/manager.py

<li>Added <code>get_log_prefix</code> function to generate log prefixes.<br> <li> Updated logging statements to include log prefixes.<br> <li> Modified function signatures to pass log prefixes.<br>

</details>

  </td>
  <td><a href="https://github.com/Significant-Gravitas/AutoGPT/pull/8001/files#diff-abdc90d8e3679ede8e93fcede5bf3a233f314c2b61aea9220007943536e7056b">+44/-15</a>&nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: rnd/autogpt_server/autogpt_server/executor/manager.py [hunk 1]
```diff
@@ -58,6 +58,10 @@ def get_log_metadata(
     }
 
 
+def get_log_prefix(graph_eid: str, node_eid: str, block_name: str = "-"):
+    return f"[ExecutionManager][graph-eid-{graph_eid}|node-eid-{node_eid}|{block_name}]"
```
Comment (majdyz) [inline rnd/autogpt_server/autogpt_server/executor/manager.py:66]: log metadata has graph-id and node-id too, can we just auto-generate the log prefix from there?

File: rnd/autogpt_server/autogpt_server/executor/manager.py [hunk 2]
```diff
@@ -124,7 +133,7 @@ def update_execution(status: ExecutionStatus):
     input_data_str = json.dumps(input_data)
     input_size = len(input_data_str)
     logger.info(
-        "Executed node with input",
+        f"{prefix}Executed node with input",
```
Comment (majdyz) [inline rnd/autogpt_server/autogpt_server/executor/manager.py]: space

File: rnd/autogpt_server/autogpt_server/executor/manager.py [hunk 3]
```diff
@@ -156,7 +165,7 @@ def update_execution(status: ExecutionStatus):
     except Exception as e:
         error_msg = f"{e.__class__.__name__}: {e}"
         logger.exception(
-            "Node execution failed with error",
+            f"{prefix}Node execution failed with error",
```
Comment (majdyz) [inline rnd/autogpt_server/autogpt_server/executor/manager.py]: space

File: rnd/autogpt_server/autogpt_server/executor/manager.py [hunk 4]
```diff
@@ -432,13 +446,14 @@ def on_node_execution(cls, q: ExecutionQueue[NodeExecution], data: NodeExecution
     def _on_node_execution(
         cls,
         q: ExecutionQueue[NodeExecution],
-        d: NodeExecution,
+        dd: NodeExecution,
```
Comment (majdyz) [inline rnd/autogpt_server/autogpt_server/executor/manager.py]: dd ?

Comment (aarushik93) [reply rnd/autogpt_server/autogpt_server/executor/manager.py]: typo

File: rnd/autogpt_server/autogpt_server/executor/manager.py (hunk 1 above)
Comment (aarushik93) [reply rnd/autogpt_server/autogpt_server/executor/manager.py:66]: I just put back in the same as what we had here previously - we can add more in, in a follow up PR for sure

**Review by majdyz — APPROVED** [review]
Thank you!

**Comment by Swiftyos** [comment]
@aarushik93 what is the status of this PR?

**Review by Swiftyos — APPROVED** [review]

**Pull request merged**